benchmark/torch/ddpg/train.py

```diff
@@ -0,0 +1,173 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import random
+import torch
+import gym
+import argparse
+from parl.utils import logger, tensorboard, ReplayMemory
+from mujoco_model import MujocoModel
+from mujoco_agent import MujocoAgent
+from parl.algorithms import DDPG
+
+
+# Run episode for training
+def run_train_episode(agent, env, rpm, args):
```

Comment:
args is a global var

Reply:
Resolved.
![image](https://user-images.githubusercontent.com/76139596/102741694-08622980-438e-11eb-9640-96f93bbd78a3.png)